README.md

```diff
@@ -1,45 +1,36 @@
 # riff Sample: Node Square
 
-A sample function that multiplies a number by itself, returning the squared value.
+A sample JavaScript function that multiplies a number by itself, returning the squared value.
 
 ## Deployment
 
-1. Setup a running riff install (tested with riff 0.0.6)
+1.  Install riff.
 
-   See riff's [Getting Started](https://github.com/projectriff/riff/blob/master/Getting-Started.adoc) guide. Skip if you already have riff running.
+    These instructions assume riff v0.2.0 or later. See riff's [Getting Started](https://projectriff.io/docs/getting-started/) guide. Skip if you already have riff running.
 
-2. Install the node invoker
+2.  Create the function.
 
-   ```bash
-    riff invokers apply -f https://github.com/projectriff/node-function-invoker/raw/v0.0.8/node-invoker.yaml
-   ```
+    This assumes that the default namespace was initialized with credentials for your `DOCKER_ID` on [Docker Hub](https://docs.docker.com/docker-hub/).
```

Comment:
if we target riff 0.3.0 we can drop the `--image` flag and this extra blurb. For riff 0.2.0 this is fine.

Reply:
this is for immediate correctness with v0.2.0 (as per PR title)